config/routes.rb

```diff
@@ -82,6 +82,7 @@
         end
       end
       resource :inactive_adviser, only: [:show]
+      resources :registered_adviser, only: [:index]
```

Comment:
@willhall88 I'm using resources because the `index` action is a plural resource that will handle generic incoming requests. This is contrary to default actions such as `show`, `new`, `create`, `destroy` and ``update` which works on single resources.